src/util/fs.js

```diff
@@ -144,21 +144,24 @@ async function buildActionsForCopy(
       srcFiles = await readdir(src);
     }
 
-    if (await exists(dest)) {
-      const destStat = await lstat(dest);
+    let destStat;
+    try {
+      destStat = await lstat(dest);
+    } catch (e) {}
```

Comment:
Oh, sure. Basically, `fs.exists` is deprecated in Node, which recommends using `fs.stat` with error handling instead. In this case `destStat` will be undefined if error occurs during `lstat`.